airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -641,7 +641,7 @@
 - name: Shopify
   sourceDefinitionId: 9da77001-af33-4bcd-be46-6252bf9342b9
   dockerRepository: airbyte/source-shopify
-  dockerImageTag: 0.1.27
+  dockerImageTag: 0.1.28
```

Comment:
Should we bump the version after publishing?

Reply:
I see, thanks for the advice.